ui/src/App.tsx

```diff
@@ -1,16 +1,24 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Switch, Route } from 'react-router-dom';
 import { Grid } from '@material-ui/core';
 import { useApi } from './lib/useApi';
 import { Spinner } from './components/Spinner';
-import { FeedbackMessage } from './components/FeedbackMessage';
+import { Notification } from './components/Notification';
+import { Slot } from './types';
 
 import { Shell } from './Shell';
-import { Slots } from './screens/Slots';
-import { Tests } from './screens/Tests';
+import { Slots as SlotsScreen } from './screens/Slots';
+import { Tests as TestsScreen } from './screens/Tests';
```

Comment:
Would it be worth defining a `./screens/index.ts` so we can collapse the imports into one like:

```
import { Tests as TestsScreen, Slots as SlotsScreen } from './screens'
```

What do you think?

Reply:
Good question. My feeling is that doing that is just as worth it as adding an index file that imports/exports all the components under `./components` and I don't think that's a very common pattern in TS projects 🤔 But I could be wrong. I propose we rename these two "screen components" explicitly to `SlotsScreen` and `TestsScreen` so at least we can do a simple named import without having to rename things.

If we do get to a point where we have many screen/page components, we may want to revisit this and add the index file you suggest, to shorten the imports block, but I don't think it's worth doing yet.